playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
I see. I have two questions about how that is working now, maybe you could help:

1. Why do we `include: ../openshift-cluster/initialize_groups.yml` in `byo/openshift-checks` and then `include: ../openshift-cluster/evaluate_groups.yml` in `common/openshift-checks`?

2. Why do we have `tags: always` in `include: ../openshift-cluster/evaluate_groups.yml`, but not in the evaluate_groups include?